parachain-template/runtime/Cargo.toml

```diff
@@ -133,16 +134,23 @@ runtime-benchmarks = [
 	'hex-literal',
 	"sp-runtime/runtime-benchmarks",
 	"xcm-builder/runtime-benchmarks",
+
 	"frame-benchmarking",
 	"frame-system-benchmarking",
 	"frame-support/runtime-benchmarks",
 	"frame-system/runtime-benchmarks",
+
 	"pallet-balances/runtime-benchmarks",
 	"pallet-collator-selection/runtime-benchmarks",
 	"pallet-template/runtime-benchmarks",
 	"pallet-timestamp/runtime-benchmarks",
 	"pallet-xcm/runtime-benchmarks",
+	"pallet-session/std",
```

Comment:
this seems wrong? this is the runtime-benchmarks section?

Reply:
err srry 🤦